code/index.html

```diff
@@ -2,11 +2,163 @@
+        <div></div>
+    </div>
+</div>
+    <div class="navigation">
+        <menu>
+            <a href="./Home.html">Home</a>
+            <a href="./Uber Uns.html">Über Uns</a>
+            <a href="./Events.html">Events</a>
+            <a href="./News.html">News</a>
+            <a href="./Inspiration & Empfehlungen.html">Inspiration & Empfehlungen</a>
+            
+                <div class="search">
+                    <input class="search-field" type="text" placeholder="Search... " name="search">
+                    <button class="search-button" type="button">🔍</button>
+                </div>  
+        
+        </menu>
+        
+    </div>
+</header>
+</head>
+<body> 
+    <section class="intro-container">
+        <div>
+            <h2>A girl gang of women agressively supporting eachother</h2>
+            <h3>In jeder von uns steckt eine Chefin</h3>
+            <p>Netzwerk von jungen, dynamischen Frauen, die sich gegenseitig unterstützen und stärken, nie ausgelernt haben und sich gerne auch mal selber feiern.</p>
+        </div>
+        <!--Kommentar für meine Übersicht oder Struktur-->
+        
+        <div>
+            <p>
+            Auf <a href="https://www.linkedin.com/groups/8967842/" target="_blank" rel="noopener noreferrer">LinkedIn</a> und WhatsApp teilen wir alles, was uns bewegt.<br>
+            <br>Wir freuen uns von dir zu hören! 
+            </p>
+    </section>
+
+    <section class="second-container">
+        <div class="second-container-left">
+            <h1>Weihnachtsfeier 2019</h1>
+            <img src="Images/Bild Weihnachtsfeier.jpg" alt="Bild von Chefinnen-Members an Weihnachtsfeier">
+        </div>
+        <div class="second-container-text">
+            <h3>Die Weihnachtsfeier 2019 war ein Highlight der Chefinnen-Events. Auch dieses Jahr wollen wir uns für ein gemütliches Abendessen wieder in den Weinnkellereien Aarau treffen. Hoffentlich kommt uns ihr-wisst-schon-was nicht in die Quere! 🤞</h3>
+        </div>
+    </section>
+
+
+    <section class="third-container">
+        <div class="Labor-Child title" id="Text">
+            <h1>Laborbesuch 2020</h1>
+            <h3>Ende Februar besuchten wir den Arbeitsplatz von Chefin Adrienne. Sie zeigte uns ihr Labor und etc etc pp. </h3>
+        </div>
+        <div class="Labor-Child img1" id="Bild1">
+             <img src="Images/Labor1.jpg" alt="Bild von Präsentation in Labor">
+         </div>
+         <div></div>
+        <div class="Labor-Child img2" id="Bild2">
+              <img src="Images/Labor2.jpg" alt="Bild von Chefin in Labor, die es selbst versucht">
+         </div>
+    </section>
+
+    <section class="Kontakt">
+        <form action="http://httpbin.org/anything" method="POST">
+            <h4>Kontaktiere Uns</h4>
+            <fieldset>     
+                    <label>
+                        <span class="labeltext">Vorname, Nachname*</span>
+                        <input type="text" name="Name" required>
```

Comment:
To make it easier to read, we can give the attributes new line if we have several of them:
```
<input
  type="text"
  name="Name"
  required>
```